Cargo.toml

```diff
@@ -1,11 +1,12 @@
 [package]
 name = "cbindgen"
-version = "0.20.0"
+version = "0.20.0-rainway.0"
```

Comment:
was thinking this might help us track what mainline version we're forked from, but definitely looking for feedback here.